pkg/vpc/vpc_test.go

```diff
@@ -56,14 +55,12 @@ var _ = Describe("VPC Endpoints", func() {
 				return input != nil
 			})).Return(mockResultFn, e.Error)
 
-			err := UseEndpointAccessFromCluster(p, cfg)
-			if err != nil {
-				Expect(err.Error() == eks.ErrCodeResourceNotFoundException)
+			if err := UseEndpointAccessFromCluster(p, cfg); err != nil {
+				Expect(err.Error()).To(Equal(eks.ErrCodeResourceNotFoundException))
```

Comment:
FTR, in the real implementation, the equality test needs to be against `awserr.Error.Code()`
```golang
awsError, ok := err.(awserr.Error)
Expect(ok).To(BeTrue())
Expect(awsError.Code()).To(Equal(eks.ErrCodeResourceNotFoundException))
```

But it's fine here since the error is set in the test case.

Reply:
Agreed :100: